cmd/ocp.go

```diff
@@ -89,13 +89,17 @@ func openShiftCmd() *cobra.Command {
 		kubeClientProvider := config.NewKubeClientProvider("", "")
 		wh = workloads.NewWorkloadHelper(workloadConfig, ocpConfig, kubeClientProvider)
 		envVars := map[string]string{
-			"ES_SERVER":     esServer,
-			"ES_INDEX":      esIndex,
-			"QPS":           fmt.Sprintf("%d", QPS),
-			"BURST":         fmt.Sprintf("%d", burst),
-			"GC":            fmt.Sprintf("%v", gc),
-			"GC_METRICS":    fmt.Sprintf("%v", gcMetrics),
-			"INDEXING_TYPE": indexer,
+			"ES_SERVER":      esServer,
+			"ES_INDEX":       esIndex,
+			"LOCAL_INDEXING": fmt.Sprintf("%v", localIndexing),
```

Comment:
```suggestion
			"LOCAL_INDEXING": fmt.Sprintf("%v", *localIndexing),
```

Reply:
We were setting a memory address in this env var LOL

Reply:
Changed as suggested. Thanks

Reply:
No, The existing snippet is correct. The `localIndexing` variable stores a pointer return by the command line flag that needs to be dereferenced in order to get its actual value. Another example [here](https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/ocp.go#L72).